### PR TITLE
3.2.0-rc.1

### PR DESCRIPTION
**Pull request by nickpatrick**
- XCFramework support
- Improve rate limit logic: In `RadarAPIHelper`, use GCD to sleep for 1 second after track requests
- Ignore client-side region transitions without a `radar_` identifier prefix
- Don't trust `CLVisit` arrivals as stops when using the responsive preset
- Add `trip` and `events` to trip callbacks
- Rename a few methods to avoid false positive App Store rejections for private APIs

### CODE REVIEW

**Comment by jsani-radar** [comment]
can you point to the reviewable portion of this PR? is it `dist/RadarSDK.xcframework/Info.plist`? hard to tell as it looks like all the model headers now are included under a directory corresponding to a specific architecture?

**Comment by nickpatrick** [comment]
@jsani-radar I'll add some instructions. Most of the relevant changes are in Xcode project files (targets and run scripts), which will be hard to read in GitHub. The files in `dist/` are the built framework files.

**Comment by nickpatrick** [comment]
I'm also going to cut an alpha release off of this branch to make sure CocoaPods and Carthage work with these changes.

File: Example/Example/AppDelegate.swift [hunk 1]
```diff
@@ -34,9 +34,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
             }
         }
 
-        let options = RadarTrackingOptions.responsive
-        options.sync = .all
-        options.showBlueBar = true
+        let options = RadarTrackingOptions.presetContinuous
```
Comment (jkao) [inline Example/Example/AppDelegate.swift:37]: I may take another pass at the example app when I have the bandwidth, but this wouldn't be launch-blocking anyways.

File: RadarSDK/RadarDelegateHolder.m [hunk 2]
```diff
@@ -49,9 +44,18 @@ - (void)didUpdateLocation:(CLLocation *)location user:(RadarUser *)user {
     if (self.delegate) {
         [self.delegate didUpdateLocation:location user:user];
     }
+
+    [[RadarLogger sharedInstance]
+        logWithLevel:RadarLogLevelInfo
+             message:[NSString stringWithFormat:@"📍 Radar location updated | coordinates = (%f, %f); accuracy = %f; link = https://radar.io/dashboard/users/%@",
```
Comment (jkao) [inline RadarSDK/RadarDelegateHolder.m:50]: nbd but I think we can grab the hostname from settings right?

File: RadarSDK/RadarRoutes+Internal.h [hunk 3]
```diff
@@ -2,7 +2,6 @@
 //  RadarRoutes+Internal.h
 //  RadarSDK
 //
-//  Created by Nick Patrick on 2/1/20.
 //  Copyright © 2020 Radar Labs, Inc. All rights reserved.
```
Comment (jkao) [inline RadarSDK/RadarRoutes+Internal.h:5]: should probably just do a big search and replace

**Review by jkao — APPROVED** [review]

**Comment by jz558611** [comment]
Will Radar be turned on again??

**Comment by jsani-radar** [comment]
hey @jz558611, Radar is always on and our APIs have a very high uptime, check out our [status page](https://status.radar.com/)

if you'd like to install Radar in your application, feel free to pull the [latest version of the iOS SDK](https://github.com/radarlabs/radar-sdk-ios/releases/tag/3.8.1)

**Pull request merged**